src/main/java/edu/hawaii/its/api/configuration/GrouperPropertyConfigurer.java

```diff
@@ -2,51 +2,35 @@
 
 import jakarta.annotation.PostConstruct;
 
-import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.core.env.Environment;
 
 import edu.internet2.middleware.grouperClient.util.GrouperClientConfig;
 
 @Configuration
 public class GrouperPropertyConfigurer {
 
-    private final Environment env;
+    @Value("${grouperClient.webService.url:}")
+    private String webServiceUrl;
 
-    @Autowired
-    public GrouperPropertyConfigurer(Environment env) {
-        this.env = env;
-    }
+    @Value("${grouperClient.webService.login:}")
+    private String webServiceLogin;
+
+    @Value("${grouperClient.webService.password:}")
+    private String webServicePassword;
```

Comment:
Is the colon ( : ) needed at the end of `@Value`? I know it can be used to define a default value when the value is not found but it seems to define an empty default value.

Reply:
That was per Frank's recommendation.  I have been consulting with him on this code since Java is not my day job and only consider chatGPT to provide suggestions.  We were trying to prevent the need to use a null guard elsewhere though I confess that I left it in anyway, just to be doubly sure.

Reply:
I note that the PR does not have periods at the ends of the sentences of the new comments.  Drat.  Will have to deal with that when the code gets revisited.

Reply:
That's totally fine, I tend to use comments like a bullet point and only use a period for a sentence but not for the very last sentence